docs/Developers/Newcomers_Guide.rst

```diff
@@ -254,16 +254,25 @@ If you know you have more work to do on this ``Pull Request`` before it is
 ready to be accepted, you can optionally indicate this to other developers
 by starting your ``Pull Request`` title with ``wip`` (case-insensitive).
 
-Step 7. Waiting for Review
+Step 7. Reviewing Process
 --------------------------
 
-After creating a Pull Request, your PR is open to the review process (to read
-more about it, have patience and it is explained on the next step), and all
-you can do is wait. The best thing you can do while at this step is review
-other people's PRs. Not only will this help the maintainers with the workload,
-but this is one of the three core steps towards becoming a full-norm coalaian.
+After creating a Pull Request, your PR is open to the review process.
 
-For more information about reviewing code, check out this `link <http://coala.io/reviewing>`_.
+..note::
+
+    You may ask for a review after you have done a review as well.
+    When you ask for it, please provide a link of something you just reviewed.
+    We need to do this so everyone gets timely reviews and people who are
+    helping with this get preferred.
+    Go to http://coala.io/review to find all PRs that are awaiting reviews.
+
+Not only will this help the maintainers with the workload,
```

Comment:
whoops this is out of context, the text above is a note and out of the text flow, this basically continues it!